docker/package/tezos_baking_wizard.py

```diff
@@ -0,0 +1,489 @@
+    }
+
+    key_import_modes = {
+        "ledger": "From a ledger",
+        "secret-key": "Either the unencrypted or password-encrypted secret key for your address",
+        "json": "Faucet JSON file from https://faucet.tzalpha.net/",
+    }
+
+    values = {}
+
+    steps = [
+        Step(
+            id="network",
+            prompt="Which Tezos network would you like to use?\nCurrently supported:\n",
+            help="The selected network will be used to set up all required services.\n"
+            "The currently supported protocol is 009-PsFLoren (used on florencenet and mainnet).\n"
+            "Also, note that the corresponding packages have protocol suffix in lowercase, "
+            "e.g. the list of available baker packages can be found on Serokell's launchpad, i.e. here:\n"
+            "https://launchpad.net/~serokell/+archive/ubuntu/tezos/+packages?field.name_filter=tezos-baker&field.status_filter=published"
+            "\nKeep in mind that you must select the test network (florencenet) if you plan "
+            "on baking with a faucet JSON file.",
+            options=pprint_table(networks),
+            validator=Validator(enum_range_validator(networks)),
+        ),
+        Step(
+            id="mode",
+            prompt="Do you want to set up baking or to run the standalone node?\n",
+            help="By default, tezos-baking provides predefined services for running baking instances "
+            "on different networks. Sometimes, however, you might want to only run the Tezos node. "
+            "When this option is chosen, this wizard will help you bootstrap tezos-node only.",
+            options=pprint_table(modes),
+            validator=Validator(enum_range_validator(modes)),
+        ),
+        Step(
+            id="snapshot",
+            prompt="The Tezos node can take a significant time to bootstrap from scratch.\n"
+            "Bootstrapping from a snapshot is suggested instead.\n"
+            "How would you like to proceed?",
+            help="A fully-synced local tezos-node is required for running a baking instance.\n"
+            "By default, service with tezos-node will start to bootstrap from scratch, "
+            "which will take a significant amount of time. In order to avoid this, we suggest "
+            "bootstrapping from a snapshot instead.\n\n"
+            "Snapshots can be downloaded from the following websites:\n"
+            "Tezos Giganode Snapshots - https://snapshots-tezos.giganode.io/ \n"
+            "XTZ-Shots - https://xtz-shots.io/ \n"
+            "We recommend to use rolling snapshots. This is the smallest and the fastest mode "
+            "that is sufficient for baking (you can read more about other tezos-node history modes here: "
+            "https://tezos.gitlab.io/user/history_modes.html#history-modes ).\n"
+            "All commands within the service are run under the 'tezos' user.",
```

Comment:
Also, probably not very useful outside of the article, as we run all of the commands ourselves.

Reply:
I don't know, the user still has to choose their preferred snapshot mode, and the info might be useful for the (maybe far-fetched) case when the download size matters. So I'd leave this in for now :shrug:

Reply:
:ok_hand: fair enough

Reply:
:ok_hand: fair enough